docs/eng-how-to.md

```diff
@@ -0,0 +1,167 @@
+# Overview
+
+The Science Design System (SDS) brings consistency and universal standards to CZI’s science products by offering a library of high quality, reusable components that deliver predictable, accessible and easy to learn experiences. Our goal is to democratize access to tools and technologies for scientists.
+
+# Getting Started
+
+## Installation
+
+NPM Package
+
+```
+// with npm
+npm i czifui
+
+// with yarn
+yarn add czifui
+```
+
+**Currently SDS uses Material UI v4**
+
+czifui installs without peer dependencies to prevent version errors. Please ensure the following dependencies are also installed:
+
+```
+  "@emotion/css"
+  "@emotion/react"
+  "@emotion/styled"
+  "@material-ui/core"
+  "@material-ui/icons"
+  "@material-ui/lab"
+  "react"
+  "react-dom"
+```
+
+To install the dependencies:
+
+```
+// with npm
+npm i @emotion/css @emotion/react @emotion/styled @material-ui/core @material-ui/icons @material-ui/lab react react-dom
+
+// with yarn
+yarn add @emotion/css @emotion/react @emotion/styled @material-ui/core @material-ui/icons @material-ui/lab react react-dom
+```
+
+Add the following HTML to your `index.html` at the `<head>` section:
```

Comment:
Since this part is already covered in the "Using default theme" section, maybe we can remove it here :D? Thank you!